code/game/machinery/turrets.dm

```diff
@@ -423,6 +423,10 @@
 		return 1
 	return
 
+/obj/machinery/turretid/AltClick(mob/user)
+	src.attackby(user.get_active_hand(), user)
```

Comment:
Isn't this just the same as clicking it normally, without holding Alt?

Reply:
Oh, this is one of those things that uses `attackby(null)` to actually do something. You should probably pass null explicitly instead of using the item in the user's active hand, which is what happens when you normal click with any item anyway

Reply:
Disregard this, apparently this already worked in Bleeding-edge. So I'll just remove this part.